website/content/commands/job/run.mdx

```diff
@@ -79,6 +79,9 @@ that volume.
 - `-preserve-counts`: If set, the existing task group counts will be preserved
   when updating a job.
 
+- `-preserve-resources`: If set, the existing task resources will be preserved
+  when updating a job.
```

Comment:
For both this feature and the `-preserve-counts` flag, it might be nice to add a note as to _why_ you'd want this flag. That is, explain that it's here to preserve these values in the face of autoscaling.

Reply:
I've added a tiny bit, but I wasn't sure how much explanation to go into based on the definitions of the other flags in this file. Is simply mentioning it what you were thinking, or were you thinking a deeper explanation? My concern is nestling away too much context into an unexpected location, but I'm open to being completely wrong about it!